test-it/ViaTests/DateFormatString.via

```diff
@@ -46,10 +46,12 @@ define(HelloTime dv(.VirtualInstrument (
 	Printf("12:(%s)\n" s1)
 	FormatDateTimeString(s1 * t1 true)
 	Printf("13:(%s)\n" s1)
+	FormatDateTimeString(s1 "%.;%x" t2 true)
+	Printf("14:(%s)\n" s1)
 	FormatDateTimeString(s1 * * true)
```

Comment:
Ah, I missed that, thanks. I think we should just then test optional argument (*) for **format string input** and **isUTC input** but with a timestamp input specified and then delete the other commented out lines.

Reply:
Fine with me to add a test which test just optional format string and isUTF inputs,
but I think we should leave the commented out tests.  Tests with commented out lines can still be useful to run by hand to sanity-check output.
For example, I did this (on purpose) for the event throughput tests.  The test just automatically verifies all events are correctly delivered without dropping any, but you can manually run it to measure how many events/second are processed (which cant be automatically checked because it varies, and if we tried to enforce some threshold range, it would be different on native vs. JS.)

Reply:
I see your point, that's fine then.